akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaRange.scala

```diff
@@ -15,14 +15,14 @@ sealed abstract class MediaRange extends jm.MediaRange with Renderable with With
   def params: Map[String, String]
   def qValue: Float
   def matches(mediaType: MediaType): Boolean
+  def isWildcard: Boolean
```

Comment:
Hah, sadly it's too strict since we need the (old code, works):

```scala
  val `*/*` = new PredefinedMediaRange("*/*") {
    def matches(mediaType: MediaType) = true
    override def isWildcard: Boolean = true
  }
  val `*/*;q=MIN` = `*/*`.withQValue(Float.MinPositiveValue)
```

to work like that in order to have the proper ordering when picking qvalues it seems...
We can live without the fancy `this ==` impl IMHO